microsoft/validation/inference_js.py

```diff
@@ -0,0 +1,290 @@
+# Authors:
+# - Nathaniel Rose <nathaniel.rose@microsoft.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+
+"""Binary for generating predictions over a set of videos. Outputs a csv or a normalized json"""
+
+import os
+import glob
+import json
+import tarfile
+import time
+import numpy
+import tensorflow as tf
+from tensorflow.python.lib.io import file_io
+from tensorflow import app
+from tensorflow import flags
+from tensorflow import gfile
+from tensorflow import logging
+
+import eval_util

```

Comment:
Can the script run from this directory? or does it need to be moved into `youtube8m`?

Reply:
No needs to be in youtube8m. Kept it here since the Keras inference isnt dependent on yt8m. What do you think?

Reply:
It's not clean to have it in this directory, but I also don't like separating this script from the Keras one :(
For now, could you just add some documentation to the top of the file noting that it needs the yt8m code to run?

Reply:
let's also rename this back to "inference_json.py". Let's also have the "main" function include a return(json) so that it displays the json output. I'm happy to make this change once it is merged as well.

Reply:
Added readme @yradsmikham Ill look to you to add the edit the script for what you need.